tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -81,14 +90,26 @@ class TestableLineChart {
 
 @Component({
   selector: 'line-chart',
-  template: '',
+  template: `
+    {{ tooltipData | json }}
+    <ng-container
+      *ngIf="tooltipTemplate"
+      [ngTemplateOutlet]="tooltipTemplate"
+      [ngTemplateOutletContext]="{data: tooltipData}"
+    ></ng-container>
+  `,
 })
 class TestableGpuLineChart {
   @Input() preferredRendererType!: RendererType;
   @Input() seriesData!: DataSeries[];
   @Input() seriesMetadataMap!: DataSeriesMetadataMap;
   @Input() yScaleType!: ScaleType;
-  @Input() ignoreYOutliers: boolean = false;
+  @Input() ignoreYOutliers!: boolean;
+  @Input()
+  tooltipTemplate!: TemplateRef<{data: TooltipDatum[]}>;
+  @Input() tooltipData: TooltipDatum[] = [];
```

Comment:
Would be worth noting that this is the only @Input() here that does not exist in the actual line-chart component. And explain briefly what it is for.